src/main/java/Duke.java

```diff
@@ -1,10 +1,74 @@
+import java.util.Scanner;
+
+/**
+ * Represents the main class for the application Duke.
+ */
 public class Duke {
```

Comment:
Do change the class name to follow your own chatbot name. You can easily do this through code refactoring.